openslides_backend/presenter/get_users.py

```diff
@@ -22,9 +38,9 @@
             "entries": {"type": "integer"},
             "sort_criteria": {
                 "type": "array",
-                "item": {
+                "items": {
                     "type": "string",
-                    "enum": ["username", "first_name", "last_name"],
+                    "enum": list(ALLOWED.keys()),
```

Comment:
Simple fix since the enum beforehand was incorrect - there even exists a test which tests that sorting by `title` works, the enum just wasn't enforced. I'm open to improving this (e.g. removing the now-redundant check of the sort criteria below), but since the presenter is not used AFAIK, I would not put any work into this. We could even consider removing it altogethere.